controller/engine_controller.go

```diff
@@ -683,7 +684,7 @@ func (ec *EngineController) stopMonitoring(engineName string) {
 
 	stopCh, ok := ec.engineMonitorMap[engineName]
 	if !ok {
-		ec.logger.WithField("engine", engineName).Warn("Stop monitoring engine called even though there is no monitoring")
+		ec.logger.WithField("engine", engineName).Warn("Stopping monitoring engine called even though there is no monitoring")
```

Comment:
I think we can delete this log which looks not really helpful.

Reply:
Removed.